--- inline/EmphStrongTrait.php	2018-03-26 19:24:36.000000000 +0800
+++ inline/EmphStrongTrait.php	2022-08-09 20:31:37.272903585 +0800
@@ -34,8 +34,7 @@
 				return [['text', $text[0] . $text[1]], 2];
 			}
 
-			if ($marker === '*' && preg_match('/^[*]{2}((?>\\\\[*]|[^*]|[*][^*]*[*])+?)[*]{2}/s', $text, $matches) ||
-				$marker === '_' && preg_match('/^__((?>\\\\_|[^_]|_[^_]*_)+?)__/us', $text, $matches)) {
+			if ($marker === '*' && preg_match('/^[*]{2}((?>\\\\[*]|[^*]|[*][^*]*[*])+?)[*]{2}/s', $text, $matches)) {
 
 				return [
 					[
@@ -54,8 +53,7 @@
 				return [['text', $text[0]], 1];
 			}
 
-			if ($marker === '*' && preg_match('/^[*]((?>\\\\[*]|[^*]|[*][*][^*]+?[*][*])+?)[*](?![*][^*])/s', $text, $matches) ||
-				$marker === '_' && preg_match('/^_((?>\\\\_|[^_]|__[^_]*__)+?)_(?!_[^_])\b/us', $text, $matches)) {
+			if ($marker === '*' && preg_match('/^[*]((?>\\\\[*]|[^*]|[*][*][^*]+?[*][*])+?)[*](?![*][^*])/s', $text, $matches)) {
 				// if only a single whitespace or nothing is contained in an emphasis, do not consider it valid
 				if ($matches[1] === '' || $matches[1] === ' ') {
 					return [['text', $text[0]], 1];
