diff --git a/inline/EmphStrongTrait.php b/inline/EmphStrongTrait.php
index ca91929..5b75c98 100644
--- a/inline/EmphStrongTrait.php
+++ b/inline/EmphStrongTrait.php
@@ -54,8 +54,7 @@ trait EmphStrongTrait
                                return [['text', $text[0]], 1];
                        }
 
-                       if ($marker === '*' && preg_match('/^[*]((?>\\\\[*]|[^*]|[*][*][^*]+?[*][*])+?)[*](?![*][^*])/s', $text, $matches) ||
-                               $marker === '_' && preg_match('/^_((?>\\\\_|[^_]|__[^_]*__)+?)_(?!_[^_])\b/us', $text, $matches)) {
+                       if ($marker === '*' && preg_match('/^[*]((?>\\\\[*]|[^*]|[*][*][^*]+?[*][*])+?)[*](?![*][^*])/s', $text, $matches)) {
                                // if only a single whitespace or nothing is contained in an emphasis, do not consider it valid
                                if ($matches[1] === '' || $matches[1] === ' ') {
                                        return [['text', $text[0]], 1];
